common/src/test/java/uk/gov/justice/services/common/converter/JsonObjectToObjectConverterTest.java

```diff
@@ -127,7 +146,7 @@ public InternalPojo getInternalPojo() {
         private UUID internalId;
         private String internalName;
 
-        public InternalPojo(final UUID internalId, final String internalName) {
+        public InternalPojo(final @JsonProperty("internalId") UUID  internalId, final @JsonProperty("internalName") String internalName) {
```

Comment:
Do we need these? The objectMapper should be able to get the names from the JDK8 parameter names.